src/main/c/internal/HybiPacketDecoder.h

```diff
@@ -32,40 +32,45 @@
 #include <string>
 #include <vector>
 
+using namespace std;
```

Comment:
Please don't use `using namespace` in a header (see [C++ CG §SF.7](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive).)